### DISCUSSION TITLE
misunderstanding concept about buffering

### DISCUSSION THREAD

**Discussion by babaie774** [learning-zone/nodejs-interview-questions#69] [General]
claim:
No Buffering – Node.js applications never buffer any data. They simply output the data in chunks.

Check:
Node.js does use buffers, which are chunks of binary data, to handle streams efficiently. Streams in Node.js can be in object mode, where each chunk is treated as a discrete piece of data, or they can be in binary mode, where data is buffered. The use of buffers allows for more efficient handling of large amounts of data, and it doesn't necessarily mean that Node.js applications never buffer any data.

**Comment by learning-zone** [comment]
Please create a Pull Request for this, thanks
